packages/esm-patient-chart-app/translations/en.json

```diff
@@ -16,7 +16,7 @@
   "checkFilters": "Check the filters above",
   "close": "Close",
   "confirm": "Confirm",
-  "confirmDeletingVisitTextWithStartAndEndDate": "Are you sure you want to delete {{visit}} which started {{visitStartDate}} and ended {{visitEndDate}}?",
+  "confirmDeletePastVisitText": "Clicking confirm will delete this {{visit}} and all of its associated encounters.",
```

Comment:
This fixes the text of the delete visit modal, which looks very awkward at the moment:

![CleanShot 2025-01-10 at 1  25 53@2x](https://github.com/user-attachments/assets/95920df7-ce7b-439d-aa61-d8ca611e1b4e)